### PR TITLE
V2: Primary Mixin

### PR DESCRIPTION
**Pull request by gvonkoss**
🎉  MY LAST `O-MAJOR` 😭

We had discussed using CSS colors instead of the palette, but we do have access to the palette and a lot of effort has gone in to making sure it is accessible, so I've decided to keep it that way for now.

Apart from introducing the primary mixin, I've removed the mixed colour that uses `sky` because it is the only colour that the internal colour palette doesn't currently support.

### CODE REVIEW

**Comment by JakeChampion** [comment]
😭

File: demos/src/bash.mustache [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <div class="demo" data-o-component='o-syntax-highlight'>
 <pre><code class='o-syntax-highlight--bash'>echo $1
-string="I'm a fan of origami."
+string="I'm a fan of Origami."
```
Comment (JakeChampion) [inline demos/src/bash.mustache:3]: 🙌

**Comment by gvonkoss** [comment]
😭

File: origami.json [hunk 2]
```diff
@@ -4,6 +4,10 @@
 	"origamiType": "module",
 	"origamiCategory": "components",
 	"origamiVersion": 1,
+	"brands": [
+		"master",
+		"internal"
+	],
```
Comment (JakeChampion) [inline origami.json:10]: 👌

**Review by JakeChampion — APPROVED** [review]

**Pull request merged**